### ISSUE TITLE
Copying a binary file to stdout fails

### ISSUE THREAD

**Issue by Philpax** [localstack/awscli-local#14]
`aws cp` supports copying to standard out through `aws cp origin -`. However, because awslocal attempts to decode the output as UTF-8, this fails for binary files:
```
> awslocal s3 cp s3://some/bucket/some_file.json.gz -
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/bin/awslocal", line 60, in output_reader
    line = to_str(line)
  File "/usr/local/bin/awslocal", line 51, in to_str
    s = s.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
```

This is easy enough to work around (creating a temporary file and then using `cat`), but it'd be great to see a fix for this if possible.

**Comment by whummer** [comment]
Thanks for reporting @Philpax . This should be fixed in 8878701d3c4e20b7b2c97985055498c71677fd35 . Can you please give it a try with the latest version `0.5` of the pip package? Please report here if the problem persists. Thanks

**Comment by Philpax** [comment]
I can confirm this works as expected now, thank you!
